inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -78,367 +78,341 @@ for (pkg in required_packages) {
 }
 
 
+
+## Set default options for plots and charts
+
 # set default text size to 16 for plots
 # give classic black/white axes for plots
 ggplot2::theme_set(theme_classic(base_size = 18))
 
-# Set the day that defines the beginning of your epiweek.
-# you can start the week on any day of the week
-# (the ISO standard is to start on Monday) 
-aweek::set_week_start("Monday")
+# sets the theme in ggplot for epicurves
+epicurve_theme <- theme(
+  axis.text.x = element_text(angle = 45, hjust = 1, vjust = 1), 
+  legend.title = element_blank(),
+  panel.grid.major.x = element_line(color = "grey60", linetype = 3),
+  panel.grid.major.y = element_line(color = "grey60", linetype = 3))
+
+# sets the labels in ggplot for the epicurves
+epicurve_labels <- labs(x = "Calendar week", 
+                        y = "Cases (n)", 
+                        title = "Cases by week of onset",
+                        subtitle = sprintf("Source: MSF data from %s", reporting_week)
+                       ) 
+
 ```
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You need to set the week you want to report on. Generally, this is the previous
+week. Put it below.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 
 ```{r define_current_week}
```

Comment:
My suggestion would be to simply merge this chunk with the one above it so that it's more clear that the above chunk is for setting up variables.

Reply:
I think it gets lost if you throw it in the setup chunk -- hard to find and realize you need to do something.